spec.html

```diff
@@ -652,28 +655,44 @@ <h1><ins>ExecuteAsyncModule ( _module_ )</ins></h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-async-module-execution-fulfilled" aoid="AsyncModuleExecutionFulfilled">
-      <h1><ins>AsyncModuleExecutionFulfilled ( _module_ )</ins></h1>
+    <emu-clause id="sec-gather-async-parent-completions" aoid="GatherAsyncParentCompletions">
+      <h1><ins>GatherAsyncParentCompletions ( _module_, _execList_ )</ins></h1>
       <emu-alg>
         1. Assert: _module_.[[Status]] is ~evaluated~.
         1. If _module_.[[AsyncEvaluating]] is *false*,
           1. Assert: _module_.[[EvaluationError]] is not *undefined*.
           1. Return *undefined*.
-        1. Assert: _module_.[[EvaluationError]] is *undefined*.
-        1. Set _module_.[[AsyncEvaluating]] to *false*.
```

Comment:
Are we no longer set asyncEvaluating to false? In this case, maybe the flag name should be changed... like "PendingAsyncEvaluation" or something...

Reply:
This was an oversight, thanks, I've added it in a follow-up iteration now. I do like the idea of `PendingAsyncEvaluation` though and that might be a possible future spec simplification if it can handle the required state checks. The `AsyncEvaluating` definition as currently is supposed to be as defined though.